From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@MisinformedDNA__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

rmcmurray commented:
This change sounds good to me.

brapel committed:
bing-custom-search: remove id from hit highlighting

jpallavi committed:
Update backup-azure-vms-encryption.md

Updated for managed BEK VM support.

jpallavi committed:
Update backup-azure-vms-automation.md

PRMerger20 committed:
Merge pull request #24865 from brapel/custom-search-page-webpages-update-sample-urls

bing-custom-search: update sample urls to show custom search

brapel committed:
bing-custom-search: add themes to hosted ui

brapel committed:
coginitive-services: updated use and display requirements

PRMerger16 committed:
Merge pull request #24899 from brapel/update-use-and-display-req

coginitive-services: updated use and display requirements

PRMerger20 committed:
Merge pull request #24898 from brapel/custom-search-hosted-ui-themes

bing-custom-search: add themes to hosted ui

brapel committed:
web-search: add entity answer

brapel committed:
use and display formatting

brapel committed:
web-search: add attribution images

PRMerger12 committed:
Merge pull request #24930 from brapel/use-and-display-two-bullets

use and display formatting

brapel committed:
bing-video-search: add code quickstarts

Ja-Dunn committed:
Merge pull request #24812 from brapel/custom-search-updated-structure

bing-custom-search: restructure

v-shils committed:
Merge pull request #24924 from brapel/web-search-add-entity-answer

web-search: add entity answer

brapel committed:
bing-web-search: add translation answer

Commit:
remove preview note from get started and overview

brapel committed:
bing-news-search: add quickstarts

brapel committed:
bing-custom-search: reorder quick-start next steps

brapel committed:
bing-custom-search: make adjust rank link

brapel committed:
bing-custom-search: update try it link

brapel committed:
bing-video-search: update see also links

brapel committed:
bing-custom-search: update hosted-ui

brapel committed:
bing-custom-search: add preview pane note, en-US market

Commit:
Migrate changes from Jerry's old working branch

brapel committed:
bing-custom-search: remove "same MSA" statement

Commit:
Add landing page icons

v-shils committed:
Merge pull request #24866 from brapel/custom-search-updat-hit-highlighting

bing-custom-search: remove id from hit highlighting

brapel committed:
bing-web-search: fix link target

PRMerger18 committed:
Merge pull request #24988 from brapel/web-search-add-translation

bing-web-search: add translation answer

fhryo-msft committed:
Add scripts in calculating container size

Add scripts in calculating container size

tynevi committed:
Merge branch 'release-bing-v7-merge' into master

tynevi committed:
removing duplicate entry

shruja committed:
SaaS App Tutorial: TextMagic

brapel committed:
bing-news-search: update TOC entries per reviewer

brapel committed:
bing-custom-search: remove id from computation image

brapel committed:
custom-search: update code samples

brapel committed:
bing-news-search: rename quick-start to quickstart

brapel committed:
bing: update use and display requirements

v-thepet committed:
Merge pull request #25101 from brapel/custom-search-remove-id-hh-computation

bing-custom-search: remove id from computation image

brapel committed:
bing-news-search: fix link

paulathurman committed:
Merge branch 'master' into sql-database-intelligent-insights-troubleshoot-performance

v-thepet committed:
Merge pull request #25103 from brapel/custom-search-code-sample-updates

custom-search: update code samples

paulathurman committed:
Update sql-database-intelligent-insights.md

Commit:
Merge pull request #25116 from brapel/update-use-and-display-requirements

bing: update use and display requirements

brapel committed:
bing-custom-search: add landing page

v-thepet committed:
Merge pull request #24990 from brapel/bing-news-search-quickstarts

bing-news-search: add quickstarts

brapel committed:
bing-custom-search: update toc to point to landing page

tynevi committed:
Merge pull request #25122 from tysonn/release-bing-v7-merge

Update release-bing-v7-merge from master, resolve conflict

v-anpasi committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into release-bing-v7-merge

brapel committed:
bing-custom-search: update header levels to conform to standards

brapel committed:
bing-custom-search: fix node.js reference

PRMerger15 committed:
Merge pull request #24995 from brapel/custom-search-adjust-rank-link

bing-custom-search: make adjust rank link

Commit:
Merge pull request #24994 from brapel/custom-search-reorder-qs-next-steps

bing-custom-search: reorder quick-start next steps

PRMerger15 committed:
Merge pull request #25149 from brapel/custom-search-consistency

bing-custom-search: update header levels to conform to standards

v-thepet committed:
Merge pull request #25151 from brapel/custom-search-nodejs-reference

bing-custom-search: fix node.js reference for consistency

v-anpasi committed:
Bringing even with upstream branch

v-anpasi committed:
Bringing even with master.

shruja committed:
update

shruja committed:
update

brapel committed:
custom-search: landing page; update author and manager

PRMerger20 committed:
Merge pull request #25030 from brapel/custom-search-remove-msa-statement

bing-custom-search: remove "same MSA" statement

Commit:
Merge pull request #25017 from brapel/custom-search-hosted-ui-feedback-updates

bing-custom-search: update hosted-ui based on feedback

PRMerger17 committed:
Merge pull request #25022 from brapel/custom-search-define-your-custom-view-specify-market

bing-custom-search: add preview pane note, en-US market

Commit:
Merge branch 'release-bing-v7-merge' of https://github.com/microsoft/azure-docs-pr into release-bing-v7-merge

Commit:
Merge pull request #24989 from swhite-msft/release-bing-v7-merge

remove preview note from get started and overview

brapel committed:
custom-search: add preview key disclaimer

Commit:
Merge pull request #25202 from brapel/custom-search-preview-key-disclaimer

custom-search: add preview key disclaimer

Lisaco88 committed:
Merge pull request #24996 from brapel/custom-search-try-it-link

bing-custom-search: update try it link

ktoliver committed:
Merge pull request #25133 from brapel/custom-search-landing-page

bing-custom-search: landing page

itechedit committed:
Merge pull request #25004 from brapel/bing-video-search-quickstarts

Bing video search quickstarts

tfitzmac committed:
added powershell quickstart

tfitzmac committed:
add ps quickstart to landing

Commit:
Final changes to address reviewer comments

v-kents committed:
updated toc/breadcrumb for quickstart links

tfitzmac committed:
fixed issues

Commit:
Update redirection file

Commit:
Fix TOC link for Bing Image Search

brapel committed:
custom-search: fix note formatting

Commit:
Merge pull request #8 from MicrosoftDocs/master

Pulling upstream

Commit:
Merge pull request #25210 from brapel/custom-search-code-sample-fix-formatting

review

Commit:
Some adjustments/fixes after Jerry's final review

Commit:
Merge branch 'release-bing-v7-merge' into release-bing-v7-merge

Commit:
give index.yml pages their own unique UUID

dlepow committed:
Batch: Updates to custom image article

dlepow committed:
edit

dlepow committed:
Acrolinx fixes

tfitzmac committed:
added portal quickstart

tdykstra committed:
clarifications

tdykstra committed:
complete sentence

Commit:
fix final blocking issues

tdykstra committed:
format a json string

Commit:
REALLY fix final blocking issue

Commit:
fix link on landing pages

Commit:
Seeded new file for product license reference content.

tfitzmac committed:
modified images

paulathurman committed:
fixing merge conflicts

paulathurman committed:
syncing changes

dlepow committed:
Updated with Donn's latest text

Commit:
First complete version of the reference doc.

paulathurman committed:
fixing merge conflicts

Commit:
Stylistic changes.

Commit:
Added a note about data freshness.

Commit:
.

Commit:
.

v-anpasi committed:
Bringing even with master.

Commit:
.

Commit:
.

Commit:
.

linda33wj committed:
Fix HTTP connector sample LS

linda33wj committed:
Update PostgreSQL connector driver version

linda33wj committed:
Increase Acrolinx score

brapel committed:
entities-search: mkt is required, remove "preview" from toc

dlepow committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into batchimage

dlepow committed:
Incorp'd Suriya's feedback

dlepow committed:
typo

brapel committed:
bing-news-video: remove preview note

brapel committed:
bing-image-search: remove "preview" from toc

brapel committed:
bing-news-search: remove "preview" from toc

brapel committed:
bing-spell-check: remove "preview" from toc

brapel committed:
bing-video-search: remove "preview" from toc

brapel committed:
bing-web-search: remove "preview" from toc

PRMerger18 committed:
Merge pull request #25284 from brapel/bing-remove-preview-note

bing-news-video: remove preview note

tynevi committed:
Merge pull request #25279 from brapel/bing-deprecate-v5

Bing remove "Preview" from toc

tynevi committed:
Merge pull request #25275 from brapel/entities-search-mkt-is-required

entities-search: mkt is required, remove "preview" from toc

Commit:
fixing mc

tdykstra committed:
cgillum feedback

jennyhunter-msft committed:
Added Signature Validation Details

- Added instructions and details about signature validation for Linux Hybrid Runbook Workers
- Added a more detailed description of the hybridgroupname parameter

tdykstra committed:
typos

jennyhunter-msft committed:
Fixed typo

- Fixed "signaure" typo

Commit:
fixing mc

Commit:
Reintroduce changes to TOCs, remove 'deprecated' wording

Commit:
fixing mc

Commit:
Merge branch 'release-bing-v7-merge' into release-bing-v7-merge

tdykstra committed:
change wording

jennyhunter-msft committed:
Fixed code snippet for signature validation

- Added missing "python" call in code snippet for signature validation

dlepow committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into batchimage

dlepow committed:
Incorp'd Donn's comments

Lisaco88 committed:
Merge pull request #25026 from jerrykindall/release-bing-v7-merge

Jerry's Bing Web Search and Bing Image Search updates

dlepow committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into batchimage

sharonlo101 committed:
added monitoring visually article

Commit:
Merge branch 'master' into 1087891

Commit:
Merge branch 'master' into 1087891

Commit:
.

Commit:
Merge branch 'release-bing-v7-merge' of https://github.com/MicrosoftDocs/azure-docs-pr into release-bing-v7-merge

sharonlo101 committed:
added toc change

sharonlo101 committed:
added Azure to title

sharonlo101 committed:
another title change

dlepow committed:
Incorp'ing Rohit's comments

paulathurman committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into sql-database-intelligent-insights-troubleshoot-performance

cherylmc committed:
update

dlepow committed:
punctuation

alkohli committed:
modified TOC for rbac in 8000 series

carolinacmoravia committed:
CC6292: Add missing href in iot-hub-java-java-process-d2c.md

This proposed file change comes from this contribution: https://github.com/MicrosoftDocs/azure-docs.de-de/pull/63/. 
Cpuld you help to review this and check if tis correct, please?
Thank you,

mimig1 committed:
Cosmos DB: Changed DocumentDB->Cosmos DB in image

dlepow committed:
typo

sethmanheim committed:
Freshness and link updates

billmath committed:
updating app proxy

paulathurman committed:
edit pass: sql-database-intelligent-insights-articles

YuriDio committed:
updating articles as per PM request

tfitzmac committed:
deleting old service principal topic

sethmanheim committed:
Freshness and link updates

alkohli committed:
added link to REST API docs

sethmanheim committed:
Fixing bad link

tfitzmac committed:
removed from toc

alkohli committed:
fixed

cherylmc committed:
update

cherylmc committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into redirer

sharonlo101 committed:
changed TOC name

cherylmc committed:
Update expressroute-troubleshooting-expressroute-overview.md

ktoliver committed:
Merge pull request #25324 from cherylmc/fixnat

update

Lisaco88 committed:
Merge pull request #24241 from v-savila/1087891

1087891

ktoliver committed:
Merge pull request #25327 from mimig1/10122017l

Cosmos DB: Changed DocumentDB->Cosmos DB in image

ktoliver committed:
Merge pull request #25238 from dlepow/batchimage

Batch: Updates to custom image article

PRMerger19 committed:
Merge pull request #25349 from cherylmc/patch-2

remove classic portal

Commit:
Fixed per v-shils review comments.

Ja-Dunn committed:
Merge pull request #25318 from sharonlo101/shlo10-12

added monitoring visually article #2

ktoliver committed:
Merge pull request #25241 from tdykstra/df4

clarifications and corrections in Durable Functions docs

Commit:
Fixed per v-shils review comments.

Commit:
Merge pull request #25271 from linda33wj/master

Update ADF copy content

ktoliver committed:
Merge pull request #25099 from shruja/textmagic

SaaS App Tutorial: TextMagic

ktoliver committed:
Merge pull request #25333 from sethmanheim/wk-trans

Freshness and link updates

ShawnJackson committed:
Merge pull request #24773 from paulathurman/sql-database-intelligent-insights-troubleshoot-performance

edit pass: Sql database intelligent insights troubleshoot performance

ktoliver committed:
Merge pull request #25342 from tfitzmac/1012certs

deleting old service principal topic

ktoliver committed:
Merge pull request #25348 from cherylmc/redirer

redirect and delete

jennyhunter-msft committed:
Modify final code snippet

- Separate final code snippet from text before and after

Lisaco88 committed:
Merge pull request #25335 from billmath/app1

updating app proxy

ktoliver committed:
Merge pull request #25341 from sethmanheim/wk-except

Freshness and link updates

SaloniSonpal committed:
Updating the next steps with all the connectivity options

Updating the next steps section with all the connectivity options. Adding pointers to connect/ query from various programming languages.

Lisaco88 committed:
Merge pull request #25297 from jennyhunter-msft/patch-3

Added Signature Validation Details

Commit:
Fixing index.yml

tynevi committed:
Merge pull request #25265 from v-jaswel/release-bing-v7-merge

Release bing v7 merge

PRMerger17 committed:
Merge pull request #25354 from SaloniSonpal/patch-3

Updating the next steps with all the connectivity options

tynevi committed:
recovering redirect item

tynevi committed:
Merge pull request #25361 from tysonn/release-bing-v7-merge

recovering redirect item

pkcsf committed:
added v 6.0

PRMerger16 committed:
Merge pull request #25367 from pkcsf/azure-doc-pkc

added v 6.0

Commit:
Fixed date format.

v-anpasi committed:
Bringing even with master.

Commit:
Added a section about conflicting service plans.

Commit:
Minor style fixes.

ranvijaykumar committed:
Added notes for Mac as review suggested row feature is not working on Mac

tfitzmac committed:
updated for issue

tfitzmac committed:
updated title

tfitzmac committed:
fixed typo for code block

genlin committed:
add more questions

genlin committed:
correct typo

Commit:
Merge pull request #25374 from ranvijaykumar/BikeShareForMac

Added notes for Mac as review suggested row feature is not working on…

genlin committed:
refine the format

genlin committed:
correct format issue

genlin committed:
correct format issue

jpallavi committed:
Update backup-azure-vms-automation.md

jpallavi committed:
Update backup-azure-vms-encryption.md

sanjeev3 committed:
Update ecommerce-retail-catalog-moderation.md

sanjeev3 committed:
Update ecommerce-retail-catalog-moderation.md

sanjeev3 committed:
Update ecommerce-retail-catalog-moderation.md

sanjeev3 committed:
Update facebook-post-moderation.md

PRMerger15 committed:
Merge pull request #25377 from sanjeev3/master

Updated Tutorial headings and descriptions

gsacavdm committed:
Fix link

MaanasSaranMSFT committed:
Update backup-azure-mabs-troubleshoot.md

Commit:
Merge pull request #25379 from MaanasSaranMSFT/patch-5

Update backup-azure-mabs-troubleshoot.md

MaanasSaranMSFT committed:
Update backup-azure-mabs-troubleshoot.md

genlin committed:
correct some typo

Commit:
Merge pull request #25381 from MaanasSaranMSFT/patch-5

Update backup-azure-mabs-troubleshoot.md

dominicbetts committed:
Change owner

PRMerger20 committed:
Merge pull request #25382 from dominicbetts/hub-owner

Change article owner

PRMerger19 committed:
Merge pull request #25326 from carolinacmoravia/patch-403

CC6292: Add missing href in iot-hub-java-java-process-d2c.md

PRMerger18 committed:
Merge pull request #24895 from JPallavi/patch-30

Update backup-azure-vms-encryption.md

linda33wj committed:
Fix blob LS sample

PRMerger17 committed:
Merge pull request #25384 from linda33wj/master

Fix blob LS sample

PRMerger12 committed:
Merge pull request #25378 from gsacavdm/patch-1

Fix link

YuriDio committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into OctoberUpdates_01

YuriDio committed:
updating articles as per PM request part 2

v-thepet committed:
Merge pull request #23983 from v-kents/vsts-linking

added links to VSTS

Lisaco88 committed:
Merge pull request #25256 from piotrci/SkuReference

New article with Id to product name mapping for licenses

v-thepet committed:
Merge pull request #25325 from alkohli/rbac

modified TOC for Rest APIs and rbac in 8000 series

YuriDio committed:
updating TOC

twooley committed:
powershell updates for 1.2.11

cherylmc committed:
Update point-to-site-how-to-radius-ps.md

Lisaco88 committed:
Merge pull request #25366 from fhryo-msft/master

Add scripts in calculating container size

tynevi committed:
Merge pull request #25372 from MicrosoftDocs/release-bing-v7-merge

Friday 9:00 am merge to master for release-bing-v7-merge branch

PRMerger19 committed:
Merge pull request #25397 from cherylmc/patch-2

Update point-to-site-how-to-radius-ps.md

Lisaco88 committed:
Merge pull request #25339 from YuriDio/OctoberUpdates_01

updating articles as per PM request

PRMerger14 committed:
Merge pull request #25395 from twooley/twpshell

powershell updates for 1.2.11

v-thepet committed:
Merge pull request #25375 from genlin/master201

storage-files-faq.md

twooley committed:
fixed capitalization in link

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

PRMerger17 committed:
Merge pull request #25400 from twooley/twpshellfix

fixed capitalization in link

v-alje committed:
Merge pull request #25401 from MicrosoftDocs/FromPublicRepo

From public repo

ktoliver committed:
Merge pull request #25215 from tfitzmac/1010psquick

added powershell quickstart
